tests/e2e/framework/util.go

```diff
@@ -161,3 +162,23 @@ func WaitForNamespacesDeleted(ctx context.Context, c clientset.Interface, namesp
 			return true, nil
 		})
 }
+
+// helmDebugLog prints the debug log of the helm client into a file in the test directory.
+func helmDebugLog(format string, v ...interface{}) {
```

Comment:
Instead of having a function here, does it make sense to store an instance to a logger that we can call. This will avoid repeatedly opening and closing the log file.

Reply:
I wanted that, but the HelmClient initialisation has a function as input 

```go
		Options: &helm.Options{
			Namespace:        f.Namespace.Name,
			RepositoryCache:  "/tmp/.helmcache",
			RepositoryConfig: "/tmp/.helmrepo",
			Debug:            true,
			DebugLog:         helmDebugLog,
		},
```

That's why I created this function, I agree an instance would be better

Reply:
We can create the instance when we instantiate the client and ensure that we close the file when the tests exit.

Note that since help uses `log.Printf` by default, we should be able to get away with calling `log.SetOutput` before instantiating the helm client.